test/syntax.js

```diff
@@ -57,63 +54,6 @@ describe('Azure swagger schema validation:', function () {
     });
   }).value();
 
-  describe('Azure composite swagger schema validation:', function () {
```

Comment:
should we be running validation on config files now instead of individual swaggers?